lib/modules/manager/pip-compile/artifacts.spec.ts

```diff
@@ -277,7 +277,6 @@ describe('modules/manager/pip-compile/artifacts', () => {
       expect(
         constructPipCompileCmd(
           Fixtures.get('requirementsNoHeaders.txt'),
```

Comment:
I don't think this check has merit. Lock files must always have a header in order to extract source files.